src/gluonts/dataset/repository/_gp_copula_2019.py

```diff
@@ -148,24 +150,23 @@ def save_metadata(
         )
 
 
-def save_dataset(dataset_path: Path, ds_info: GPCopulaDataset):
+def save_dataset(
+    dataset_path: Path, ds_info: GPCopulaDataset, ds_writer: DatasetWriter
```

Comment:
I think we should be consistent. Let's use either `writer`, `ds_writer` or `dataset_writer` -- but only one of them.